pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -547,6 +547,14 @@ func buildServiceTags(metadata *model.NodeMetadata, labels map[string]string) []
 				},
 			},
 		},
+		{
+			Tag: "istio.cluster_id",
+			Type: &tracing.CustomTag_Environment_{
+				Environment: &tracing.CustomTag_Environment{
```

Comment:
Why not obtain this statically similar to the variables above like metadata.ClusterID?